inc/saiport.h

```diff
@@ -2128,6 +2367,10 @@ typedef struct _sai_port_api_t
     sai_get_port_pool_stats_fn        get_port_pool_stats;
     sai_get_port_pool_stats_ext_fn    get_port_pool_stats_ext;
     sai_clear_port_pool_stats_fn      clear_port_pool_stats;
+    sai_create_port_serdes_fn         create_port_serdes;
```

Comment:
-> sai_create_port_lane_fn

Reply:
hi,
I have questions about usage of  typedef sai_status_t (*sai_create_port_serdes_fn)( ).
What attributes should be provided to create serdes object?   Specify switch_id,  SAI_PORT_SERDES_ATTR_PORT_ID, and what else?  
We know a port has a number of serdeses, why the output parameter is only one serdes ID instead of serdes ID list?

BR.
@dgsudharsan 
@lguohan

Reply:
> hi,
> I have questions about usage of typedef sai_status_t (*sai_create_port_serdes_fn)( ).
> What attributes should be provided to create serdes object? Specify switch_id, SAI_PORT_SERDES_ATTR_PORT_ID, and what else?
> We know a port has a number of serdeses, why the output parameter is only one serdes ID instead of serdes ID list?
> 
> BR.
> @dgsudharsan
> @lguohan

Hi Brandon,
If you check the attributes are per port and not per lane. The attributes are set as a list for a port(list size equal to number of lanes). Hence we would be having only one serdes attribute per port.

Reply:
Hi dgsudharsan,
In your opinion, a lane and serdes aren't similar things?
Each port has only one serdes object ID representing/including a number of lanes, right?

Reply:
Hi dgsudharsan,
If execute sai_remove_port_serdes_fn)() to remove the serdes objecet for a port, what will happen?
Does it mean this port has no serdes to use and this port's status is down?